packages/dapi-grpc/protos/platform/v0/platform.proto

```diff
@@ -1001,3 +1002,92 @@ message GetPathElementsResponse {
     GetPathElementsResponseV0 v0 = 1;
   }
 }
+
+message GetStatusRequest {
+  message GetStatusRequestV0 {
+  }
+
+  oneof version { GetStatusRequestV0 v0 = 1; }
+}
+
+
+message GetStatusResponse {
+  message GetStatusResponseV0 {
+    message Version {
+      message Software {
+        string dapi = 1;
+        string drive = 2;
+        string tenderdash = 3;
+      }
+
+      message Protocol {
+        message Tenderdash {
+          int32 p2p = 1;
+          int32 block = 2;
+        }
+
+        message Drive {
+          int32 max = 3;
+          int32 current = 4;
+        }
+
+        Tenderdash tenderdash = 1;
+        Drive drive = 2;
+      }
+
+      Software software = 1;
+      Protocol protocol = 2;
+    }
+
+    message Time {
+      uint64 local = 1;
+      uint64 block = 2;
+      uint32 epoch = 3;
+    }
+
+    message Node {
+      string id = 1;
```

Comment:
it's always named node_id, I would stick to it. I know it's inside node ;), but still